libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/pivotTableHasTotalAndSubtotalsOnDashboard.spec.ts

```diff
@@ -0,0 +1,58 @@
+// (C) 2023 GoodData Corporation
+import * as Navigation from "../../tools/navigation";
+import { Export } from "../../tools/export";
+import { TopBar } from "../../tools/dashboards";
+import { Widget } from "../../tools/widget";
+
+const widget = new Widget(0);
+const topBar = new TopBar();
+const exportControl = new Export();
+
+describe("Export dashboard", { tags: ["checklist_integrated_tiger"] }, () => {
+    beforeEach(() => {
+        Navigation.visit("dashboard/dashboard-pivot-table-scenario");
+        widget.waitTableLoaded();
+    });
+
+    it("should export insight to CSV from dashboards", () => {
+        topBar.dashboardTitleExist().dashboardTitleHasValue("Pivot Table Dashboard");
+        exportControl.exportInsightOnWidgetByIndexToCSV(0);
+        exportControl.expectExportedCSV(
+            "Table has AM metric.csv",
+            "WonderKid,89,41055,41144,60,41055,41115,188,41055,41243,123165,123502,68,41055,41123,32,41055,41087,84,41055,41139,123165,123349,246330,246851\n",
+        );
+    });
+
+    it.skip("should export insight to XLSX from dashboards", () => {
```

Comment:
For the skip test, please add this line before the skip test and the reason why we skip this test, for example:
// eslint-disable-next-line jest/no-disabled-tests
// This test is blocked by this ticket: <input the ticket number here>